### ISSUE TITLE
Two questions about softmax

### ISSUE THREAD

**Issue by FightForCS** [ShuangLI59/person_search#103]
Hi, thanks for your work. I have two questions about softmax loss in your paper

1. softmax classifier suffers from biased gradient, but it can be sloved by using larger batch size. Have you ever tried this setting? (Since in experiments part, you have compared softmax and OIM, but you did not mention the batch size for training)

2. How to 'proper pretraining' for softmax? Is it possible to pretrain for OIM loss?

Thank you.
